record_tag_helper.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir['CHANGELOG.md', 'LICENSE', 'README.md', 'lib/**/*']
 
-  spec.add_dependency 'rails', '~> 5.0.0.alpha'
+  spec.add_dependency 'rails', '>= 5.0.0.beta1', '< 6.0'
```

Comment:
Yeah, so I guess we should just remove the minimum Ruby version constraint in this gemspec.